docs/guides/contexts_expressions.md

```diff
@@ -396,7 +396,7 @@ Out[12]: 1
    context menu might update the items in the menu that are visible and/or enabled:
 
    ```python
-   context_menu.update_from_context(get_context(layer_list))
+   context_menu.update_from_context(get_context(layer-list))
```

Comment:
Is this correct? doesn't look pythonic. Also not related to this PR...

Reply:
Oh i bet it was search-replace!

Reply:
Thanks, fixed